packages/fxa-payments-server/src/components/PlanDetails/index.test.tsx

```diff
@@ -221,6 +221,52 @@ describe('PlanDetails', () => {
     });
   });
 
+  it('does not render a single tax of 0 amont', async () => {
+    updateConfig({
+      featureFlags: {
+        useStripeAutomaticTax: true,
+      },
+    });
+    (apiInvoicePreview as jest.Mock).mockClear().mockResolvedValue({
+      ...INVOICE_PREVIEW_EXCLUSIVE_TAX,
+      tax: [
+        {
+          amount: 0,
+          inclusive: false,
+          display_name: 'Sales Tax',
+        },
+      ],
+    });
+    const props = {
+      ...{
+        profile: userProfile,
+        showExpandButton: false,
+        isMobile: false,
+        selectedPlan,
+      },
+    };
+    const subject = () => {
+      return render(<PlanDetails {...props} />);
+    };
+
+    const { queryByTestId } = subject();
+
+    const formattedExpectedAmount = formatPlanPricing(
+      INVOICE_PREVIEW_EXCLUSIVE_TAX.total,
+      selectedPlan.currency,
+      selectedPlan.interval,
+      selectedPlan.interval_count
+    );
+
+    await waitFor(() => {
+      const totalPriceComponent = queryByTestId('total-price');
+      expect(totalPriceComponent).toBeInTheDocument();
+      expect(totalPriceComponent?.innerHTML).toContain(formattedExpectedAmount);
```

Comment:
Sneaky, sneaky... What does the `?.innerHTML` do?
If the previous line (L263) asserts that `totalPriceComponent` is in the document, wouldn't we know it's already truthy/valid?

Reply:
this was copied from the test above. ive removed it from this test since its unnecessary for the purpose of this test